clients/admin-ui/src/features/system/VendorSelector.tsx

```diff
@@ -130,165 +132,194 @@ const VendorSelector = ({
+        </HStack>
+        <Box width="100%" data-testid="input-name">
+          <CreatableSelect
+            name="name"
+            options={suggestions}
+            isRequired
+            value={selected}
+            onBlur={handleBlur}
+            onChange={(newValue, actionMeta) =>
+              handleSelectChange(newValue, actionMeta)
+            }
+            onInputChange={(e) => setSearchParam(e)}
+            inputValue={searchParam}
+            size="sm"
+            onKeyDown={(e) => {
+              if (e.key === "Tab") {
+                handleTabPressed();
+              }
+            }}
+            tabSelectsValue={hasVendorSuggestions}
+            classNamePrefix="custom-select"
+            placeholder="Enter system name..."
+            instanceId="select-name"
+            isDisabled={!isCreate && lockedForGVL}
+            menuPosition="absolute"
+            isSearchable
+            isClearable={!!searchParam}
+            focusBorderColor="primary.600"
+            formatCreateLabel={(inputValue) =>
+              `Create new system "${inputValue}"...`
+            }
+            chakraStyles={{
+              container: (provided) => ({
+                ...provided,
+                flexGrow: 1,
+                backgroundColor: "white",
+              }),
+              option: (provided, state) => ({
+                ...provided,
+                background:
+                  state.isSelected || state.isFocused ? "gray.50" : "unset",
+              }),
+              menu: (provided) => ({
+                ...provided,
+                visibility: hasVendorSuggestions ? "visible" : "hidden",
+              }),
+              dropdownIndicator: (provided) => ({
+                ...provided,
+                display: "none",
+              }),
+              indicatorSeparator: (provided) => ({
+                ...provided,
+                display: "none",
+              }),
+            }}
+            components={{ Option: CustomDictOption }}
+          />
+        </Box>
+        <ErrorMessage
+          isInvalid={isInvalid}
+          message={meta.error}
+          fieldName="name"
         />
-      </HStack>
-      {fieldsSeparated ? (
-        <DictSuggestionTextInput
-          id="name"
-          name="name"
-          dictField={(vendor) => vendor.name ?? (vendor.legal_name || "")}
-          isRequired
-          label="System name"
-          tooltip="Give the system a unique, and relevant name for reporting purposes. e.g. “Email Data Warehouse”"
+        <Text
+          aria-hidden
+          style={{
+            position: "absolute",
+            backgroundColor: "transparent",
+            borderColor: "transparent",
+            marginTop: "31.52px",
+            marginLeft: "13px",
+            pointerEvents: "none",
+            fontSize: "14px",
+            zIndex: 1,
+          }}
+        >
+          <span style={{ color: "transparent" }}>{searchParam}</span>
```

Comment:
I know you just moved this around from existing code, but it is weird to me that we aren't using chakra components here (and for the `<Text/>` that we are using, right above, we are passing `style` as opposed to using the chakra provided classnames